### ISSUE TITLE
target_dir with spaces

### ISSUE THREAD

**Issue by niro1987** [thomasmauerer/hassio-addons#7]
Setting `target_dir` to something like `Home Assistant/Snapshots` is not possible as quotation marks are automatically deleted on the configuration page.

```
[16:27:57] WARNING: cd \Home\: NT_STATUS_OBJECT_NAME_NOT_FOUND
[16:27:57] FATAL: Target directory does not exist. Please check your config.
```

![before save](https://user-images.githubusercontent.com/7474625/85924541-5987d980-b893-11ea-9013-88c855bd8d92.png)
![after save](https://user-images.githubusercontent.com/7474625/85924544-5c82ca00-b893-11ea-867f-e2d84aeed7ff.png)

**Comment by niro1987** [comment]
same behavior with double quotes

**Comment by thomasmauerer** [comment]
Yes, you're right - thanks for the report. Will fix that.

**Comment by thomasmauerer** [comment]
Fixed in v2.4
